x/chainlink/client/cli/queryFeed.go

```diff
@@ -7,21 +7,27 @@ import (
 	"github.com/cosmos/cosmos-sdk/client"
 	"github.com/cosmos/cosmos-sdk/client/flags"
 	"github.com/spf13/cobra"
+	"strconv"
 )
 
-func CmdListFeed() *cobra.Command {
+func CmdGetRoundFeedData() *cobra.Command {
```

Comment:
nit but maybe use `CmdGetFeedDataByRound`

Reply:
yeah whatever name is ok to me, I simply follow the method name from proposal : `getRoundData` and `latestRoundData`, to be more precisely, the data here is the `feedData`, in case we have some other `data` which I think it's highly likely would be. so `CmdGetFeedDataByRound` works for me